examples/03_LLD_CoRh2O4.jl

```diff
@@ -26,13 +26,17 @@ sys = resize_supercell(sys, (10, 10, 10))
 @assert energy_per_site(sys) ≈ -2J*S^2
 
 # Use the stochastic Landau-Lifshitz dynamics to thermalize system into
-# equilibrium at finite temperature. This is a [`Langevin`](@ref) equation,
-# which includes damping and noise terms. The strength of the noise term is
-# automatically fixed according to the damping time scale `λ` and the target
-# temperature, according to a fluctuation-dissipation theorem.
-
-Δt = 0.05/abs(J*S)   # Time step
-λ  = 0.1             # Dimensionless damping time-scale
+# equilibrium at finite temperature via the [`Langevin`](@ref) dynamics. The
+# timestep ``Δt`` controls integration accuracy. In `:dipole` mode, it should be
+# inversely proportional to the largest effective field in the system. For
+# CoRh₂O₄, this is the antiferromagnetic exchange ``J`` times the spin magnitude
+# ``S=3/2``. The dimensionless parameter ``λ`` determines the magnitude of
+# Langevin noise and damping terms. A reasonable choice is `λ = 0.2`. The
+# temperature `kT` is linked to the magnitude of the Langevin noise term via a
+# fluctuation-dissipation theorem.
+
+Δt = 0.05/abs(J*S)   # Integration timestep
+λ  = 0.2             # Dimensionless damping time-scale
```

Comment:
What was the motivation for taking λ to 0.2? More reasonable decorrelation times? (I think I used 0.2 for FeI2.)

Reply:
It is problem dependent, but I suspect that larger λ speeds decorrelation in the sense that "gradient descent" becomes more efficient. But in other ways it could slow things, because you're damping out spin waves that may carry information ballistically.